chisel.yaml

```diff
@@ -1,13 +1,51 @@
 format: v1
 
 archives:
+  # archive.ubuntu.com/ubuntu/
```

Comment:
minor note: this is for amd64 and i386 archs as other archs use the ports URL

Reply:
Indeed! Thank you. Updated it.